src/test/java/org/junit/runner/RequestTest.java

```diff
@@ -0,0 +1,21 @@
+package org.junit.runner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+public class RequestTest {
```

Comment:
You need to add RequestTest.class to src/test/java/org/junit/runner/AllRunnerTests.java in order for the Maven build to run your test.

Please verify the test fails when the changes you made to Computer are removed